components/listingsPage/dummy.js

```diff
@@ -32,3 +37,89 @@ export const Comms = styled.span`
   vertical-align: super;
   display: inline-block;
 `;
+
+export const FilterFormFIeld = styled(Formfield)`
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding-bottom: 20px;
+  border-bottom: 1px solid #edf0f0;
+  padding-left: 0;
+`;
+export const ExtHeading = styled(Heading)`
+  margin-top: 0;
+  margin-bottom: 0;
+  background: #003d8e;
+  color: #fff;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  padding: 12px 18px;
+  display: flex;
+  flex-direction: row;
+  flexwrap: wrap;
+  justify-content: space-between;
+  ${resolveMedia.md`
+   background: #edf0f0;
+    color: #515151
+`};
+`;
+
+export const ToggleIcon = styled.span`
```

Comment:
I think this element would be better as a `button`: it would be focusable and therefore better from an accessibility POV, though it also feels more semantic. If it's easier this could be considered under a accessibility card rather than this one.

Reply:
I think you are quite right, most especially from accessibility POV but i will like to leave it to be considered under an accessibility card.

Reply:
Makes sense to me!